project/ProjectPlugin.scala

```diff
@@ -29,7 +29,7 @@ object ProjectPlugin extends AutoPlugin {
       val log4s: String         = "1.8.2"
       val logback: String       = "1.2.3"
       val monix: String         = "3.2.1"
-      val mu: String            = "0.22.0"
+      val mu: String            = "0.22.3"
```

Comment:
Could we also bump `sbt-mu-srcgen` in the plugins.sbt file?

Reply:
Can do!  Good callout.